hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -28,69 +34,144 @@
  * @author Jay Shaughnessy
  * @author Lucas Ponce
  */
-public class Trigger extends TriggerTemplate {
+public class Trigger {
```

Comment:
+1,TriggerTemplate was an original guess at what we may need for a template level but it was not needed and this is definitely more clean.